hypervisor/dm/vpci/pci_pt.c

```diff
@@ -293,7 +292,33 @@ void init_vdev_pt(struct pci_vdev *vdev)
 			}
 		}
 	}
+}
+
+/*
+ * @brief Initialize a specified passthrough vdev structure.
+ *
+ * The function init_vdev_pt is used to initialize a vdev structure. If a vdev structure supports
+ * SRIOV capability that the vdev represents a SRIOV physical function(PF) virtual device, then
+ * function init_vdev_pt can initialize PF vdev SRIOV capability if parameter is_pf_vdev is true.
+ *
+ * @param vdev        pointer to vdev data structure
+ * @param is_pf_vdev  indicate SRIOV capability initialization
```

Comment:
No. This is to indicate the 1st parameter vdev is an data structure of a PF, which contains the SR-IOV capability.

Reply:
Get it, I will fix it